src/components/05_pages/NodeEditForm/index.js

```diff
@@ -77,12 +81,34 @@ styles = {
   `,
 };
 
+const extractRestorableEntity = (state, entity) => {
+  const restorableEntity = state.content.restorableContentEditById[entity.id];
+  if (
+    // @todo maybe that's too defensive?
```

Comment:
I think this looks fine to me. I'm wondering if we should have some kind of helper function for this type of conditions.

Reply:
Yeah I guess we could have one of those. The same probably for setters.